### DISCUSSION TITLE
I need another way to sign to Student

### DISCUSSION THREAD

**Discussion by Yggdrrasil** [community/community#139969] [GitHub Education]
### Hi everyone,

Hi! i would like to know if there is another way to apply to github student

I'm in Spain doing an official course by
https://www.ilerna.es/
but they don't provide .edu email

is there any other way to apply if there is no .edu email?

**Comment by akeelshafi** [comment]
Yes, you can still apply for the GitHub Student Developer Pack without a .edu email! Here's how:

1. Use School ID or Enrollment Proof: GitHub allows students to submit alternative forms of proof. You can upload:
   - A school ID card.
   - A transcript.
   - A letter of enrollment from your institution.

2. Alternative Email Address: If your school provides any official email that isn’t a .edu domain (like one from *ilerna.es*), you can still use it for verification.

3. Submit Documentation: When applying for the Student Developer Pack, you'll see an option to submit documentation instead of using an .edu email. Use this to upload your proof of enrollment.

Make sure the documents clearly show your name and the institution's name.

**Reply by akeelshafi** [reply]
If you find this solution helpful, please mark it as answered. 😊

**Reply by joshmzamora** [reply]
It requires a verified email before you can continue with the application process. I don't have an .edu email, but I do have a school ID card and a transcript. However, there is no place for me to upload it without having a verified school email.

**Reply by ArjunCodess** [reply]
+1
